code/__DEFINES/rust_g_overrides.dm

```diff
@@ -1,3 +1,3 @@
 // RUSTG_OVERRIDE_BUILTINS is not used since the file APIs don't work well over Linux.
-#define url_encode(text) rustg_url_encode("[text]")
+//#define url_encode(text) rustg_url_encode("[text]") // This is slower than the DM one
 #define url_decode(text) rustg_url_decode("[text]")
```

Comment:
I did remove this one as it was not tested, but I know for a fact the rust encode is slower.